extensions/amp-a4a/0.1/amp-a4a.js

```diff
@@ -463,6 +466,13 @@ export class AmpA4A extends AMP.BaseElement {
           if (!fetchResponse || !fetchResponse.arrayBuffer) {
             return null;
           }
+          // If the response has response code 204, collapse it.
+          if (fetchResponse.status == 200) {
```

Comment:
Also, please move this block after the `this.protectedEmitLifecycleEvent_('adRequestEnd');`, below.  We want to get the ping whenever the request returns, regardless of what its return value is.